### DISCUSSION TITLE
Application Update Notification [ WCFM - MV Fluxstore ]

### DISCUSSION THREAD

**Discussion by nikhilbhoyar** [inspireui/support#5375] [Working on Features]
When Application, new update is release at playstore then 
Application Update In-APP Notification should be shown to user.

This is use-full as if we release any major app update,
but previously app downloaded user remains on older version only without this force to update notification.

And moreover, it will be great if you allow us to choose from below options for our app users :

1. Force update option [ Update Application or close application ]
2. Later update option [ Update application now or Update Later ]

_Reference :_ _https://pub.dev/packages/in_app_update_

**Comment by simonpham** [comment]
Force update is not supported in [in_app_update](https://pub.dev/packages/in_app_update).

Consider using another package for force update option: [https://pub.dev/packages/upgrader](https://pub.dev/packages/upgrader)

**Comment by osman-kocak** [comment]
i think this option will keep the user from applying and spooking them 
Later update option [ Update application now or Update Later ]

**Comment by HanielQC** [comment]
Hi guys, Version Update Alert feature is available from 3.0.5 and later. More detail in the ChangeLog https://www.notion.so/FluxStore-WooCommerce-591ca13557434bd88a9163e617d5eebd

Without purchase, you can view a demo App of your Website with the drag&drop FluxBuilder https://fluxbuilder.com for Free 🔻
🎯 Edit your App's UI by drag&drop 
🎯 Build a Test APK to view how your App looks
Quick guide: https://www.notion.so/FluxBuilder-s-guide-a5164230ff444676b10b5cadc30302aa

Very Best!
